### DISCUSSION TITLE
What is the concept of work distribution between workers?

### DISCUSSION THREAD

**Discussion by Mich341** [coinfoundry/miningcore#1201] [Q&A]
Hey! after a new block is found, the same task is sent to each worker, and when the resolved shares are returned, does the pool check them by itself? Or each different depending on their power?

**Comment by Mich341** [comment]
> Hey! after a new block is detected, the same task is sent to each worker, and when the resolved shares are returned, does the pool check them by itself? Or each different depending on their power?

**Comment by XIII39** [comment]
Every GPU receives the same work.

A very simplified way to think about it is:
"Hey GPUs, if I give you these 2 number can you guess what the third number is to make a solution that is the right difficulty or higher?"

If a GPU finds a solution that it close to or higher than what would be considered a block solution, the pool submits it to the node for consideration.

**Comment by seperatis** [comment]
Not an issue, please close or move.
